enterprise/frontend/src/embedding-sdk/components/private/AppInitializeController.tsx

```diff
@@ -23,14 +26,33 @@ export const AppInitializeController = ({
   useInitData({ config });
 
   const isInitialized = useSdkSelector(getIsInitialized);
+  const licenseProblem = useSdkLicenseProblem(config);
+
+  const content = useMemo(() => {
+    if (!isInitialized) {
+      return <SdkLoader />;
```

Comment:
I noticed we were using the `Loading...` text, so might be a good idea to use the loader component - this falls back to the loading indicator, which would look nicer than text.